Improve fides.js installation doc with suggested test steps, modal link instructions, etc. (#199)
ethyca/fidesdocs: pages/tutorials/consent-management-configuration/install-fides.mdx

2. Optionally, remove the `<script>Fides.gtm()</script>` tag if you do not use Google Tag Manager for managing tags on your website

<Callout type="info" emoji="">
##### Installation Location
It's important that Fides.js is loaded as early as possible on your site. For this reason, it's recommended to place these tags ahead of all other JavaScript in the `<head>` of your website.
</Callout>

### Test Fides.js Installation

Once you have installed the `fides.js` script and deployed the changes to your website, you can do some quick tests before fully configuring your consent solution. Here are three easy checks:
1. Open your updated website in your web browser, e.g. https://www.example.com
2. Inspect the source code of your website and ensure that the `<script src="https://privacy.example.com/fides.js"></script>` is present
3. Open the Developer Console and run `window.Fides` and ensure that you see a valid Javascript object like:
```
{consent: {…}, experience: undefined, geolocation: undefined, options: {…}, fides_meta: {…}, …}
```

If `window.Fides` is loading, you're almost done!

## Step Two: Add a Link or Button to Your Website Footer

> To make it possible for your website visitors to manage their consent preferences directly on your website, you can add a new link or button to your website to show the consent overlay. You're probably familiar with this technique - often, this will be a link or button in the footer like "Manage Preferences", "Your Privacy Choices", or similar.

Reviewer: ```suggestion
For full consent compliance, it's often necessary to have a link or button on your site so that your website visitors can directly manage their consent preferences. Commonly, this will be a link or button in the footer like "Manage Preferences", "Your Privacy Choices", or similar. You can either edit your existing element or add a new element on your page show the Fides consent overlay by using `id="fides-modal-link"` on that element:
```

```html filename="Fides.js Modal Link"
<body>
  <!-- Step Two: Add a custom link/button/etc. to the footer of your site with id="fides-modal-link" -->
  <a href="#" id="fides-modal-link">
    Manage Preferences
  </a>
</body>
```

<Callout type="info" emoji="">
##### Supported Elements

> Technically, Fides.js registers an `onclick` handler on any clickable element: `<a>`, `<button>`, `<div>`, etc.
> </Callout>

Reviewer: ```suggestion
In the above example, we use an `<a>` element, but you can add the `fides-modal-link` id on any clickable element: `<a>`, `<button>`, `<div>`, etc. Regardless of which you use, Fides.js will register an `onclick` handler on this element.
</Callout>
```

Once you have added this link or button and deployed the changes to your website, Fides.js will automatically look for the element by searching for `id="fides-modal-link"`.

> You should be able to click the link and open the Fides overlay! However, there's one catch - Fides will automatically hide the link if there are no relevant notices in the user's location!

Reviewer: ```suggestion
You should be able to click the link and open the Fides overlay! Note that the link is hidden by default, and will only show if there are relevant notices in the user's location.
```

> You should be able to click the link and open the Fides overlay! However, there's one catch - Fides will automatically hide the link if there are no relevant notices in the user's location!

Reviewer 2: Should we provide additional steps to force the behavior needed for testing? Or leave this up to FDE/CS to do this in the near-term?

Author: Right- that's a good suggestion. I'm going to merge this version as-is and there are two follow-ups:
1. How to "force" the banner for simplified testing
2. How to style the banner with a custom stylesheet
